wenet/transformer/decoder.py

```diff
@@ -169,3 +178,153 @@ def forward_one_step(
         if self.use_output_layer:
             y = torch.log_softmax(self.output_layer(y), dim=-1)
         return y, new_cache
+
+class BidirectionalDecoder(TransformerDecoder):
```

Comment:
rename to BiTransformerDecoder in case that we have other type of decoder in the future.